
时间：2017-03-16 11:12:37,022 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：15484 
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 系统找不到指定的文件。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   在 System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   在 System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.Table`1.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   在 System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source, Expression`1 predicate)
   在 X.App.Views.mgr.xmg.InitDict() 位置 D:\Pro\x.yfj\src\Web\Yfj\X.App\Views\mgr\xmg.cs:行号 56
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.yfj\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.yfj\src\Web\X.Web\XApp.cs:行号 31
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:2,State:0,Class:20

时间：2017-03-16 11:43:24,099 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：814 
NVelocity.Exception.ParseErrorException: Encountered "," at line 92, column 36.
Was expecting one of:
    <RPAREN> ...
    <WHITESPACE> ...
    "-" ...
    "+" ...
    "*" ...
    "/" ...
    "%" ...
    "&&" ...
    "||" ...
    "<" ...
    "<=" ...
    ">" ...
    ">=" ...
    "==" ...
    "!=" ...
    <DOT> ...
    "," ...
    "=" ...
     ---> NVelocity.Runtime.Parser.ParseException: Encountered "," at line 92, column 36.
Was expecting one of:
    <RPAREN> ...
    <WHITESPACE> ...
    "-" ...
    "+" ...
    "*" ...
    "/" ...
    "%" ...
    "&&" ...
    "||" ...
    "<" ...
    "<=" ...
    ">" ...
    ">=" ...
    "==" ...
    "!=" ...
    <DOT> ...
    "," ...
    "=" ...
    
   在 NVelocity.Runtime.Parser.Parser.Parse(TextReader reader, String templateName)
   在 NVelocity.Runtime.RuntimeInstance.Parse(TextReader reader, String templateName, Boolean dumpNamespace)
   在 NVelocity.Runtime.RuntimeInstance.Parse(TextReader reader, String templateName)
   在 NVelocity.Template.Process()
   --- 内部异常堆栈跟踪的结尾 ---
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.yfj\src\Web\X.Web\Views\View.cs:行号 108
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.yfj\src\Web\X.Web\XApp.cs:行号 31

时间：2017-03-16 11:43:52,762 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：29476 
NVelocity.Exception.ParseErrorException: Encountered "#end\r\n" at line 111, column 5.
Was expecting one of:
    <EOF> 
    "(" ...
    <RPAREN> ...
    <ESCAPE_DIRECTIVE> ...
    <SET_DIRECTIVE> ...
    "\\\\" ...
    "\\" ...
    <TEXT> ...
    <SINGLE_LINE_COMMENT> ...
    "*#" ...
    "*#" ...
    <STRING_LITERAL> ...
    "if" ...
    "stop" ...
    <NUMBER_LITERAL> ...
    <WORD> ...
    <IDENTIFIER> ...
    <DOT> ...
    "{" ...
    "}" ...
     ---> NVelocity.Runtime.Parser.ParseException: Encountered "#end\r\n" at line 111, column 5.
Was expecting one of:
    <EOF> 
    "(" ...
    <RPAREN> ...
    <ESCAPE_DIRECTIVE> ...
    <SET_DIRECTIVE> ...
    "\\\\" ...
    "\\" ...
    <TEXT> ...
    <SINGLE_LINE_COMMENT> ...
    "*#" ...
    "*#" ...
    <STRING_LITERAL> ...
    "if" ...
    "stop" ...
    <NUMBER_LITERAL> ...
    <WORD> ...
    <IDENTIFIER> ...
    <DOT> ...
    "{" ...
    "}" ...
    
   在 NVelocity.Runtime.Parser.Parser.Parse(TextReader reader, String templateName)
   在 NVelocity.Runtime.RuntimeInstance.Parse(TextReader reader, String templateName, Boolean dumpNamespace)
   在 NVelocity.Runtime.RuntimeInstance.Parse(TextReader reader, String templateName)
   在 NVelocity.Template.Process()
   --- 内部异常堆栈跟踪的结尾 ---
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.yfj\src\Web\X.Web\Views\View.cs:行号 108
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.yfj\src\Web\X.Web\XApp.cs:行号 31

时间：2017-03-16 11:44:11,663 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：48378 
NVelocity.Exception.ParseErrorException: Encountered "#end\r\n" at line 111, column 5.
Was expecting one of:
    <EOF> 
    "(" ...
    <RPAREN> ...
    <ESCAPE_DIRECTIVE> ...
    <SET_DIRECTIVE> ...
    "\\\\" ...
    "\\" ...
    <TEXT> ...
    <SINGLE_LINE_COMMENT> ...
    "*#" ...
    "*#" ...
    <STRING_LITERAL> ...
    "if" ...
    "stop" ...
    <NUMBER_LITERAL> ...
    <WORD> ...
    <IDENTIFIER> ...
    <DOT> ...
    "{" ...
    "}" ...
     ---> NVelocity.Runtime.Parser.ParseException: Encountered "#end\r\n" at line 111, column 5.
Was expecting one of:
    <EOF> 
    "(" ...
    <RPAREN> ...
    <ESCAPE_DIRECTIVE> ...
    <SET_DIRECTIVE> ...
    "\\\\" ...
    "\\" ...
    <TEXT> ...
    <SINGLE_LINE_COMMENT> ...
    "*#" ...
    "*#" ...
    <STRING_LITERAL> ...
    "if" ...
    "stop" ...
    <NUMBER_LITERAL> ...
    <WORD> ...
    <IDENTIFIER> ...
    <DOT> ...
    "{" ...
    "}" ...
    
   在 NVelocity.Runtime.Parser.Parser.Parse(TextReader reader, String templateName)
   在 NVelocity.Runtime.RuntimeInstance.Parse(TextReader reader, String templateName, Boolean dumpNamespace)
   在 NVelocity.Runtime.RuntimeInstance.Parse(TextReader reader, String templateName)
   在 NVelocity.Template.Process()
   --- 内部异常堆栈跟踪的结尾 ---
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.yfj\src\Web\X.Web\Views\View.cs:行号 108
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.yfj\src\Web\X.Web\XApp.cs:行号 31

时间：2017-03-16 11:44:25,427 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：62142 
NVelocity.Exception.MethodInvocationException: Invocation of method 'get_goods' in  X.App.Views.wx.index threw exception System.ArgumentNullException : 值不能为 null。
参数名: text ---> System.ArgumentNullException: 值不能为 null。
参数名: text
   在 System.Data.Linq.SqlClient.SqlHelpers.GetStringContainsPattern(String text, Char escape, Boolean& usedEscapeChar)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.TranslateStringMethod(SqlMethodCall mc)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitMethodCall(SqlMethodCall mc)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitBinaryOperator(SqlBinary bo)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitBinaryOperator(SqlBinary bo)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitBinaryOperator(SqlBinary bo)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitBinaryOperator(SqlBinary bo)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSequence(SqlSelect sel)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExists(SqlSubSelect sqlExpr)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSubSelect(SqlSubSelect ss)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Convert(SqlNode node, SqlFactory sql, ProviderMode providerMode)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(ResultShape resultShape, Type resultType, SqlNode node, ReadOnlyCollection`1 parentParameters, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(Expression query, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Views.wx.index.get_goods(String cate, Int32 top) 位置 D:\Pro\x.yfj\src\Web\Yfj\X.App\Views\wx\index.cs:行号 34
   --- 内部异常堆栈跟踪的结尾 ---
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.yfj\src\Web\X.Web\Views\View.cs:行号 108
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.yfj\src\Web\X.Web\XApp.cs:行号 31

时间：2017-03-16 11:45:36,183 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：23459 
NVelocity.Exception.MethodInvocationException: Invocation of method 'get_goods' in  X.App.Views.wx.index threw exception System.ArgumentNullException : 值不能为 null。
参数名: text ---> System.ArgumentNullException: 值不能为 null。
参数名: text
   在 System.Data.Linq.SqlClient.SqlHelpers.GetStringContainsPattern(String text, Char escape, Boolean& usedEscapeChar)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.TranslateStringMethod(SqlMethodCall mc)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitMethodCall(SqlMethodCall mc)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitBinaryOperator(SqlBinary bo)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitBinaryOperator(SqlBinary bo)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitBinaryOperator(SqlBinary bo)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitBinaryOperator(SqlBinary bo)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSequence(SqlSelect sel)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExists(SqlSubSelect sqlExpr)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSubSelect(SqlSubSelect ss)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitExpression(SqlExpression exp)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitAlias(SqlAlias a)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSource(SqlSource source)
   在 System.Data.Linq.SqlClient.SqlVisitor.VisitSelectCore(SqlSelect select)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Visitor.VisitSelect(SqlSelect select)
   在 System.Data.Linq.SqlClient.SqlVisitor.Visit(SqlNode node)
   在 System.Data.Linq.SqlClient.PostBindDotNetConverter.Convert(SqlNode node, SqlFactory sql, ProviderMode providerMode)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(ResultShape resultShape, Type resultType, SqlNode node, ReadOnlyCollection`1 parentParameters, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(Expression query, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Views.wx.index.get_goods(String cate, Int32 top) 位置 D:\Pro\x.yfj\src\Web\Yfj\X.App\Views\wx\index.cs:行号 33
   --- 内部异常堆栈跟踪的结尾 ---
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.yfj\src\Web\X.Web\Views\View.cs:行号 108
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.yfj\src\Web\X.Web\XApp.cs:行号 31

时间：2017-03-16 14:06:32,797 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：795 
NVelocity.Exception.ParseErrorException: Encountered "() {\r\n                " at line 269, column 36.
Was expecting one of:
    "," ...
    ")" ...
     ---> NVelocity.Runtime.Parser.ParseException: Encountered "() {\r\n                " at line 269, column 36.
Was expecting one of:
    "," ...
    ")" ...
    
   在 NVelocity.Runtime.Parser.Parser.Parse(TextReader reader, String templateName)
   在 NVelocity.Runtime.RuntimeInstance.Parse(TextReader reader, String templateName, Boolean dumpNamespace)
   在 NVelocity.Runtime.RuntimeInstance.Parse(TextReader reader, String templateName)
   在 NVelocity.Template.Process()
   --- 内部异常堆栈跟踪的结尾 ---
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.yfj\src\Web\X.Web\Views\View.cs:行号 108
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.yfj\src\Web\X.Web\XApp.cs:行号 31

时间：2017-03-16 15:32:27,287 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T用户登陆超时)","code":"T用户登陆超时"} 
时间：80328 

时间：2017-03-16 15:36:16,409 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T商品存不存在或已经下架)","code":"T商品存不存在或已经下架"} 
时间：6505 

时间：2017-03-16 15:36:49,577 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"errcode":40163,"errmsg":"code been used, hints: [ req_id: zNTE0a0809ssz2 ]"} 
时间：39673 

时间：2017-03-16 15:36:57,679 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T商品存不存在或已经下架)","code":"T商品存不存在或已经下架"} 
时间：47775 

时间：2017-03-16 15:38:49,706 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"errcode":40163,"errmsg":"code been used, hints: [ req_id: Ic1KLA0929ssz2 ]"} 
时间：159803 

时间：2017-03-16 15:38:57,827 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"errcode":40163,"errmsg":"code been used, hints: [ req_id: IC3FmA0937ssz1 ]"} 
时间：167923 

时间：2017-03-16 15:39:30,692 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T商品存不存在或已经下架)","code":"T商品存不存在或已经下架"} 
时间：200788 

时间：2017-03-16 15:39:36,489 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"errcode":40163,"errmsg":"code been used, hints: [ req_id: _dqUma0976ssz2 ]"} 
时间：206585 

时间：2017-03-16 15:39:38,759 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"errcode":40163,"errmsg":"code been used, hints: [ req_id: GKRE0a0978hsz3 ]"} 
时间：208856 

时间：2017-03-16 15:39:42,694 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"errcode":40163,"errmsg":"code been used, hints: [ req_id: zxqWwa0982sz11 ]"} 
时间：212790 

时间：2017-03-16 15:41:00,903 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"errcode":40163,"errmsg":"code been used, hints: [ req_id: SVOiYa0060ssz2 ]"} 
时间：621 

时间：2017-03-16 15:41:04,158 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"errcode":40163,"errmsg":"code been used, hints: [ req_id: AfiuGa0064ssz2 ]"} 
时间：3877 

时间：2017-03-16 15:41:42,720 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"errcode":40163,"errmsg":"code been used, hints: [ req_id: IHiFOA0102hsz5 ]"} 
时间：42438 

时间：2017-03-16 15:58:02,738 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T商品存不存在或已经下架)","code":"T商品存不存在或已经下架"} 
时间：2562 

时间：2017-03-16 15:58:05,898 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T商品存不存在或已经下架)","code":"T商品存不存在或已经下架"} 
时间：5721 

时间：2017-03-16 16:00:31,381 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T商品在此城市无货)","code":"T商品在此城市无货"} 
时间：207 

时间：2017-03-16 16:19:04,171 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T商品存不存在或已经下架)","code":"T商品存不存在或已经下架"} 
时间：813821 

时间：2017-03-16 16:40:51,479 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T商品存不存在或已经下架)","code":"T商品存不存在或已经下架"} 
时间：720018 

时间：2017-03-16 17:33:56,245 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：92387 
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   在 System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   在 System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.Table`1.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   在 System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source, Expression`1 predicate)
   在 X.App.Views.wx._wx.InitView() 位置 D:\Pro\x.yfj\src\Web\Yfj\X.App\Views\wx\_wx.cs:行号 105
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.yfj\src\Web\X.Web\Views\View.cs:行号 94
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.yfj\src\Web\X.Web\XApp.cs:行号 31
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20

时间：2017-03-16 17:33:56,245 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：92387 
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   在 System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   在 System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.Table`1.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   在 System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source, Expression`1 predicate)
   在 X.App.Views.wx._wx.InitView() 位置 D:\Pro\x.yfj\src\Web\Yfj\X.App\Views\wx\_wx.cs:行号 105
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.yfj\src\Web\X.Web\Views\View.cs:行号 94
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.yfj\src\Web\X.Web\XApp.cs:行号 31
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20

时间：2017-03-16 17:33:56,245 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(D:\Pro\x.yfj\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：92387 
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   在 System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   在 System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.Table`1.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   在 System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source, Expression`1 predicate)
   在 X.App.Views.wx._wx.InitView() 位置 D:\Pro\x.yfj\src\Web\Yfj\X.App\Views\wx\_wx.cs:行号 105
   在 X.Web.Views.View.GetResponse() 位置 D:\Pro\x.yfj\src\Web\X.Web\Views\View.cs:行号 94
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.yfj\src\Web\X.Web\XApp.cs:行号 31
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
